### PR TITLE
Remove all references to vision

### PR DESCRIPTION
**Pull request by RealOrangeOne**
This competition, we're not shipping vision at all. To remove confusion, we should remove all references to it. 

After the competition, this PR can simply be reverted to restore the content.

### CODE REVIEW

**Comment by PeterJCLaw** [comment]
I suggest removing the changelog as it no longer means anything -- I've done this in #145.

File: content/tutorials/kit-assembly.md [hunk 1]
```diff
@@ -58,7 +57,6 @@ Make sure to read all these **before** you start assembly.
 6. The Arduino Servo Shield should already be connected into the top of the Arduino, but, if not, the pins on the bottom fit directly into the sockets with the same labels on the top of the Arduino.
 7. Connect the Arduino to the Pi by way of the beige USB-A to USB-B cable; the square-ish USB-B end goes into the appropriate metal-cased connector on the Arduino, the standard USB end goes into any free USB port on the Pi or connected USB hub. This will provide both power and communication to the Arduino.
 8. Connect the Arduino Servo Shield to the Power Board by screwing the green connector on the top and the small (3.81mm) CamCon provided onto the opposite ends of a pair of wires and plugging the CamCon into the second of the 5V power rail sockets on the Power Board.
-9. Connect the USB Webcam to the Pi via any one of the remaining USB sockets on the Pi or connected USB hub.
 10. To connect the motors, first screw wires connected to their pins into one of the medium-sized (5mm) CamCons and plug these into either of the two appropriate sockets on a Motor Board.
```
Comment (trickeydan) [inline content/tutorials/kit-assembly.md]: We should probably update the numbering here

**Comment by PeterJCLaw** [comment]
I suggest merging `master` outwards to this branch (it now contains #145, which should fix the broken links issue)

**Comment by trickeydan** [comment]
Not sure why I marked this PR as Frozen, I presume it was an accident

File: content/api/vision/_index.md [hunk 2]
```diff
@@ -1,31 +0,0 @@
...
-
-{{% notice tip %}}
-A marker's position can be represented using both the [Cartesian](coordinates/#cartesian-coordinates) and [spherical](coordinates/#spherical-coordinates) coordinate systems.
-{{% /notice %}}
-
-## Marker PDFs
-You can download PDF files of all markers:
-
-- [Large markers](/docs/large-tags.pdf), Containing Wall and Column markers (250x250mm, should be printed on A3)
-- [Small markers](/docs/small-tags.pdf), Containing Token markers (100x100mm, should be printed on A4)
```
Comment (PeterJCLaw) [inline content/api/vision/_index.md]: We should also remove these linked files too.

**Pull request merged**